Scripts/typo3-console.php

```diff
@@ -15,8 +15,8 @@
     }
     if (!file_exists(dirname($vendorAutoLoadFile) . '/composer/platform_check.php')) {
         // Do our own basic platform check, when the Composer generated one is not available
-        if (PHP_VERSION_ID < 70200) {
-            echo 'This version of TYPO3 Console requires PHP 7.2.0 or above!' . PHP_EOL;
+        if (PHP_VERSION_ID < 70400) {
```

Comment:
Ah, good catch. This is an old check, that meanwhile can and should be removed, since typo3 console now requires composer runtime api.

If you'd change this PR to only remove this check, I'd be happy to merge it. 

I'll open the PR for that purpose

Reply:
done